webrtc.html

```diff
@@ -1639,10 +1639,42 @@
                 which point the <code><a>RTCPeerConnection</a></code> can fully
                 adopt the pending local description, or rollback to the current
                 description if the remote side rejected the change.</p>
-                <p>When the method is invoked, the User Agent MUST return the
-                result of <a href="#set-description">setting the
-                RTCSessionDescription</a> indicated by the method's first
-                argument.</p>
+                <p>As noted in <span data-jsep="modifying-sdp">[[!JSEP]]</span>
+                the SDP returned from <code>createOffer</code> or
+                <code>createAnswer</code> MUST NOT be changed
+                before passing it to <code>setLocalDescription</code>. As
+                a result, when the method is invoked, the User Agent MUST
+                run the following steps:
+                <ol>
+                  <li>Let <var>description</var> be the argument to
+                  <code>setLocalDescription</code>.</li>
+                  <li>Let <var>lastOffer</var> be the result returned by the
+                  last call to <code>createOffer</code>.</li>
+                  <li>Let <var>lastAnswer</var> be the result returned by the
+                  last call to <code>createAnswer</code>.</li>
+                  <li>If <code><var>description</var>.sdp</code> is
+                  <code>null</code> and <code><var>description</var>.type</code>
+                  is <code>answer</code> or <code>pranswer</code>,
+                  set <code><var>description</var>.sdp</code> to
+                  <var>lastAnswer</var>.</li>
+                  <li>If <code><var>description</var>.sdp</code> is
+                  <code>null</code> and <code><var>description</var>.type</code>
+                  is <code>offer</code>, set <code><var>description</var>.sdp</code>
+                  to <var>lastOffer</var>.</li>
+                  <li>If <code><var>description</var>.type</code> is
+                  <code>offer</code> and <code><var>description</var>.sdp</code>
+                  does not exactly match <var>lastOffer</var>,
```

Comment:
What went into JSEP isn't that the SDP must be *exactly* the same, but that the parsed representation must be the same. For example, "a=" lines appearing in a different order shouldn't matter.

Maybe we could simplify this by just referring to JSEP:

> If `description.sdp` is not `null`, and the browser determines that it was modified after the last call to `createOffer`/`createAnswer`, as described by JSEP Section 5.8, reject `p` with an `InvalidAccessError` and abort these steps.